packages/medusa/src/api/routes/store/carts/update-cart.ts

```diff
@@ -104,6 +110,18 @@ export default async (req, res) => {
     cartDataToUpdate.billing_address = billing_address
   }
 
+  if (typeof validated.sales_channel_id !== "undefined") {
```

Comment:
thought(non-blocking): Would this be more appropriate to have as either a middleware or part of the `CartService.update` method. What do you think?

Reply:
I think that all of this a previous processing should go to the service. Should that be included as part of that PR?

Reply:
Yeah, that would be great to keep the handler as clean as possible !